src/ru/multimanga/src/eu/kanade/tachiyomi/extension/ru/multimanga/MultiManga.kt

```diff
@@ -0,0 +1,442 @@
+package eu.kanade.tachiyomi.extension.ru.multimanga
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import org.jsoup.select.Elements
+import java.text.SimpleDateFormat
+
+class MultiManga : ParsedHttpSource() {
+    override val name = "Multi-manga"
+    override val baseUrl = "https://w1.multi-manga.com"
+    override val lang = "ru"
+    override val supportsLatest = true
+    private val dateFormat: SimpleDateFormat = SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
+
+    // similar/modified theme of "https://komikindo.id"
+
+    // Formerly "Bacakomik" -> now "BacaKomik"
+    // override val id = 4383360263234319058
+
+    override val client: OkHttpClient = network.cloudflareClient.newBuilder()
+        .rateLimit(12, 3)
+        .build()
+
+    override fun popularMangaRequest(page: Int): Request {
+        return GET("$baseUrl", headers)
+    }
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        return GET("$baseUrl", headers)
+    }
```

Comment:
there is only like 5 items you are selecting from the front page for popular, it would be better to set `supportslatest` to false and return the whole page through popular alone

also, add the next page to the request, because currently you are requesting the same page over and over again